google/cloud/bigquery/schema.py

```diff
@@ -77,6 +86,9 @@ def __init__(
         description=_DEFAULT_VALUE,
         fields=(),
         policy_tags=None,
+        precision=_DEFAULT_VALUE,
+        scale=_DEFAULT_VALUE,
+        maxLength=_DEFAULT_VALUE,
```

Comment:
Didn't the liner complain here about a camelCase name?

Reply:
Apparently not. :)

Which is a good thing, since I have no control over this name.  This is the parameter name defined by BQ.

Reply:
Yeah, the field names in API response use camel case, thus there exist several `_PROPERTY_TO_API_FIELD` mappings in various classes. These mappings are used to get obtain the actual key to use when looking up the properties in API dicts.

Unless I'm missing something, it should be possible to rename the `maxLength` kwarg to `max_length`.

Reply:
Interesting.  I'm worried about users reading about `maxLength` somewhere, but I'll change this.  It should only affect `from_api_repr` and `to_api_repr` (at least after I removed some duplicate logic).